mflowgen/common/soc-rtl-v2/configure.yml

```diff
@@ -25,3 +25,7 @@ commands:
 
 parameters:
   include_core: 1
+  soc_only: undefined
+  INCLUDE_XGCD: undefined
+  TLX_FWD_DATA_LO_WIDTH: undefined
+  TLX_REV_DATA_LO_WIDTH: undefined
```

Comment:
these params actually need to be part of the synthesis node, since that's where this script is run, but I can add that now